server/block/shulker_box_types.go

```diff
@@ -0,0 +1,205 @@
+package block
+
+// ShulkerBoxType represents a type of shulker box.
```

Comment:
It would be nice if we could avoid this. Candles & bundles will also have a no color option.